### PR TITLE
fix url for templates

### PR DESCRIPTION
**Pull request by maxhansson**
wip - first fix for #162

### CODE REVIEW

File: app/views/accounts/_list.html.haml [hunk 1]
```diff
@@ -3,7 +3,8 @@
   = link_to "#{t(:disable)} #{t(:accounts)}", accounting_plan_disable_accounts_path(@accounting_plan), class: 'btn btn-success', method: 'post', disabled: !@accounting_plan.disable_accounts?
 
 .col-sm-40.col-md-40.pull-right.text-right
-  = form_tag accounting_plan_path(@accounting_plan), class: 'form-inline submit-on-change', method: :get do
+  = form_tag accounting_plan_path(@accounting_plan), class: 'form-inline submit-on-change', method: :get,  enforce_utf8: false  do
```
Comment (qzio) [inline app/views/accounts/_list.html.haml]: why `enforce_utf8: false` ? shouldn't this be true?

Comment (maxhansson) [inline app/views/accounts/_list.html.haml]: With enforce_utf8 true or without the parameter the utf8 is added to to url.

Comment (qzio) [inline app/views/accounts/_list.html.haml]: but that's ok no?

Comment (maxhansson) [inline app/views/accounts/_list.html.haml]: The problem: "When selecting in a list the resulting url is "utf8=" insted of "locale=""

Comment (qzio) [inline app/views/accounts/_list.html.haml]: hm, but it should be both, no?

Anyways, lets merge this as-is and fix/change if appropriate.

**Pull request merged**